codalab/worker_manager/aws_batch_worker_manager.py

```diff
@@ -98,6 +98,8 @@ def start_worker_job(self):
             command.extend(['--max-work-dir-size', self.args.worker_max_work_dir_size])
         if self.args.worker_delete_work_dir_on_exit:
             command.extend(['--worker-delete-work-dir-on-exit'])
+        if self.args.exit_after_num_runs and self.args.exit_after_num_runs > 0:
```

Comment:
Are exit_after_num_runs and worker_exit_after_num_runs the same? Ditto for slurm batch manager